RIT-MDRC/Catbot
# Feat/cli

# Features
* Failed to add Arduino-Cli integration due to incompatibility with RaspberryPi and Arduino-Cli
* Add clear direction on how to set up Arduino extension in VSCode
* Setup arduino.json
* Setup gitignore
* Add sample blink code
* Set up repository to follow basic community standard
   - Add Code of conduct
   - Add Contributing
   - Add Security Policy
   - Add Pull Request Template
   - Add Issue Template

# How to Test
* Follow the directions on the ReadMe file and make sure all of the steps are clearly written
* Check to see if there are any errors in the main .ino file
* Make sure the main .ino file passes the verification of the Arduino extension

merged with #18
